AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+/// A component that provides a form for Online Banking payment.
+public final class OnlineBankingComponent: PaymentComponent,
+                                           PresentableComponent,
+                                           LoadingComponent,
+                                           PaymentAware {
+
+    /// Configuration for Online Banking Component.
+    public typealias Configuration = BasicComponentConfiguration
+
+    /// The context object for this component.
+    @_spi(AdyenInternal)
+    public var context: AdyenContext
+
+    public var paymentMethod: PaymentMethod { onlineBankingPaymentMethod }
+
+    public weak var delegate: PaymentComponentDelegate?
+
+    public lazy var viewController: UIViewController = SecuredViewController(child: formViewController,
+                                                                             style: configuration.style)
+
+    public var requiresModalPresentation: Bool = true
+
+    /// Component's configuration
+    public var configuration: Configuration
+
+    private let onlineBankingPaymentMethod: OnlineBankingPaymentMethod
+
+    private var selectedIssuer: Issuer
+
+    // MARK: - Items
+
+    private var termsAndConditionsLink: String {
+        return paymentMethod.type == .onlineBankingCZ ? "https://static.payu.com/sites/terms/files/payu_privacy_policy_cs.pdf" : "https://static.payu.com/sites/terms/files/payu_privacy_policy_sk.pdf"
+    }
+
+    /// The terms and condition message item.
+    // swiftlint:disable line_length
```

Comment:
If you break lines after each parameter like after `localizationParamters` too, this warning may go away (also if you change the `termAndConditionText` to generic name as I mentioned above, that would help as well).